### DISCUSSION TITLE
Updating Windows Alternative Setup?

### DISCUSSION THREAD

**Discussion by PaxVobiscuit** [Taxel/PlexTraktSync#1303] [Q&A] [answered]
Is it possible to "selfupdate" the Windows Alternative Setup below?  If so, how?  If not, what is the correct process to update it?

https://github.com/Taxel/PlexTraktSync#windows-setup-optional-alternative

Thanks in advance!

**Comment by tangcla** [comment]
Curious to know as well.
I have in the past copied the files from a new release tag to the folder, but this seems to break the existing installation.

**Reply by KoenBoone** [reply]
I am curious to know as well :-)

**Comment by PaxVobiscuit** [comment]
@glensc Would you be the person who could tell us how to update this without a complete reinstall?

**Reply by tangcla** [reply]
> @glensc Would you be the person who could tell us how to update this without a complete reinstall?

Not me :) I want to know.

**Reply by glensc** [reply]
I don't do windows and never will.

- https://www.youtube.com/watch?v=daPI9nLr3-M

**Reply by glensc** [reply]
the logic should be something like:
- rename old folder
- install to the same folder, don't forget dependencies!
- copy config and cache from old folder

**Comment by glensc** [comment, marked as answer]
you should just install with pipx if you want self-update
